nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/TestHttpServer.cs

```diff
@@ -69,6 +75,105 @@ public static TestHttpServer CreateTestStringServer(Func<string, (int, string)>
             return CreateTestServer(bytesRequestHandler);
         }
 
+        public static TestHttpServer CreateTestNuGetFeed(params MockNuGetPackage[] packages)
```

Comment:
We commonly need to create test-only HTTP NuGet feeds.  In the case where the feed is expected to be well-behaved, this helper makes it a one-line addition to a test.